src/data/eventSchedule.js

```diff
@@ -54,54 +53,70 @@ const eventSchedule = [
 				name: 'Meme Competition Opens',
 				duration: '',
 				location: 'Discord: #memes',
-				description: `Whether you're attending the hackathon in-person or online,
-				drop your best meme in the #memes channel and get a raffle entry
-				for participating! This is an all-day event, so take your time in coming up
-				with that singular meme-tastic work of art.`
+				description: `Drop your best meme in the #memes channel on Discord and 
+                get a raffle entry for participating! This is an all-day event, so take 
+                your time in coming up with that singular meme-tastic work of art.`
 			},
 			{
 				name: 'Team Formation',
 				duration: '30 min',
-				location: 'Covel Grand Horizon/Discord',
+				location: 'Carnesale Commons Palisades Room',
 				description: `Looking for a team? Find one to hack with during
 				our team building session!`
 			}
 		]
 	},
 	{
-		startTime: getDateObj('26', '11:30:00'),
+		startTime: getDateObj('05', '09:30:00'),
+		events: [
+			{
+				name: 'Common Experience Workshop',
+				duration: '30 min',
+				location: 'Carnesale Commons Hermosa Room',
+				description: `Learn more about the Common Experience theme of intersectional 
+                environmentalism and brainstorm project ideas! You'll receive raffle entries for
+                attending as well :)`
+			}
+		]
+	},
+	{
+		startTime: getDateObj('05', '12:30:00'),
 		events: [
 			{
 				name: 'Lunch',
 				duration: '60 min',
-				location: 'Covel Grand Horizon Terrace',
+				location: 'Carnesale Commons Palisades Room',
 				description: `Coding is done best on a full stomach.
-				Come one, come all—we feast ;)! For lunch, we'll be serving In-n-Out with
+				Come one, come all—we feast! For lunch, we'll be serving In-n-Out with
 				alternate options if you have dietary restrictions. Do feel free to
 				continue working through lunch, though. The only thing better than
 				getting free food is having coding epiphanies while eating the free food.`
-			},
+			}
+		]
+	},
+	{
+		startTime: getDateObj('05', '13:00:00'),
+		events: [
 			{
-				name: 'JEDI Allyship Space',
-				duration: '45 min',
-				location: 'Covel Grand Horizon',
-				description: `We'll be having a guided discussion about topics
-				related to the Netflix documentary 'The Social Dilemma'
-				and the ethics of technology from the morality of data collection
-				to the environmental impact of tech to algorithmic bias in AI.
-				Get a raffle entry just for attending and a second one for participating.
-				Sounds like a bargain, no?`
+				name: 'Interdisciplinary Research Panel',
+				duration: '60 min',
+				location: 'Carnesale Commons Palisades Room',
+				description: `We have an amazing panel of UCLA professors/researches whose
```

Comment:
Wait is it supposed to be 'professors/researches' or 'professors/researchers'?

Reply:
oops it's supposed to be researchers, just fixed